contrib/upgrade_cypress_30.sh

```diff
@@ -30,6 +30,16 @@ function pull_git_tag() {
     # pull in case we're on a branch
     sudo -u cypress git pull
     sudo -u cypress git stash pop
+    rc=$?
+    if [[ $rc != 0 ]]; then
+      sudo -u cypress cp config/cypress.yml config/cypress.yml.old
+      echo "WARNING: Unable to merge cypress config cleanly, you will need to manually reset your cypress settings."
```

Comment:
Might make sense to point them directly at the file they need to change, in addition to where they can find the old version